templates/default/vlan/vlanlist.html

```diff
@@ -55,7 +55,7 @@ <h1>{$pagetitle|escape}</h1>
             </td>
             <td class="buttons">
                 {button type="link" icon="edit" href="?m=vlanlist&action=modify&id={$v.id}" tip="Edit"}
-                {button type="link" icon="delete" href="?m=vlanlist&action=delete&id={$v.id}" tip="Delete" id="delete-vlan"}
+                {button type="link" icon="delete" href="?m=vlanlist&action=delete&id={$v.id}" tip="Delete" class="delete-vlan"}
```

Comment:
No widzisz - gdy dodałeś więcej vlanów niż jeden to od razu zauważyłeś warning w konsoli przeglądarki www? :)

Reply:
Takie - wady instancji testowej na localhost.